win32ss/user/ntuser/desktop.c

```diff
@@ -43,17 +43,18 @@ IntFreeDesktopHeap(IN PDESKTOP pdesk);
 #ifdef _WIN64
 DWORD gdwDesktopSectionSize = 20 * 1024; // 20 MB (Windows 7 style)
 #else
-DWORD gdwDesktopSectionSize = 3 * 1024; // 3 MB (Windows 2003 style)
+DWORD gdwDesktopSectionSize = 3 * 1024;  // 3 MB (Windows 2003 style)
 #endif
 DWORD gdwNOIOSectionSize    = 128;
 DWORD gdwWinlogonSectionSize = 128;
 
-/* Currently active desktop */
-PDESKTOP gpdeskInputDesktop = NULL;
+PDESKTOP gpdeskInputDesktop = NULL;     ///< Currently active desktop.
 HDC ScreenDeviceContext = NULL;
 PTHREADINFO gptiDesktopThread = NULL;
 HCURSOR gDesktopCursor = NULL;
 PKEVENT gpDesktopThreadStartedEvent = NULL;
+PKEVENT gpDesktopSwitchEvent = NULL;    ///< WinSta0_DesktopSwitch event.
```

Comment:
Is there something depending on this name?
If not, then it might make sense to indicate that it is a legacy thing.